metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/rest_commands.py

```diff
@@ -58,17 +74,94 @@ def init_kafka_acls(self):
 
     def start_rest_application(self):
         Logger.info('Starting REST application')
-        command = format("service metron-rest start {metron_jdbc_password!p}")
-        Execute(command)
+
+        # Build the spring options
+        # the vagrant Spring profile provides configuration values, otherwise configuration is provided by rest_application.yml
+        metron_spring_options = format(" --server.port={metron_rest_port}")
```

Comment:
This is probably just a style issue and not critically important but would consider moving some of this logic to a start script that lives inside the metron-rest module?  MPack code is notoriously challenging to troubleshoot and having all this in here will make starting the REST app outside of Ambari very tedious.  I have had to do this a lot when troubleshooting REST problems and helping others.  Maybe REST is fairly stable at this point and we shouldn't have to mess with it much though.

This would also keep you from having to add a metron_version property to Ambari because the version would be substituted in the script at build time.  This is how all our other scripts get the version.

Reply:
@merrimanr So I tried doing this, but the problem is that if the Java command gets kicked up from a script, the Java command ends up showing up on in ps, which then goes and gives up spring.datasource.password.  Which is incredibly annoying.

Unless you have any ideas on how to get around that, I think we're stuck in Ambari

Reply:
Ignore the things I say, it would have helped if I'd done stuff right.  PR is updated to primarily run from a script, that's largely a renamed and slightly modified metron-rest (rename is to metron-rest.sh).  Need to update docs to reflect it.

I also killed the metron version variable by doing some shell globbing.  If you have multiple metron-rest jars in the lib directory it could be problematic, but I don't think it's unreasonable to not add random crap to the dirs.